### PR TITLE
[WIP] Add support for podman cp --follow-link

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
podman cp --follow-link is now support, Tells Podman to de-reference symbolic links before copying them.
```

### CODE REVIEW

File: cmd/podman/containers/cp.go [hunk 1]
```diff
@@ -93,6 +95,12 @@ func cp(cmd *cobra.Command, args []string) error {
 		return copyFromContainer(sourceContainerStr, sourcePath, destPath)
 	}
 
+	if followLink {
```
Comment (vrothberg) [inline cmd/podman/containers/cp.go:98]: Does it also apply to symlink _inside_ a container?

Comment (rhatdan) [reply cmd/podman/containers/cp.go:98]: I don't know.

**Comment by rhatdan** [comment]
Fixes: https://github.com/containers/podman/issues/16585

**Comment by vrothberg** [comment]
I think we can close this PR in favor of https://github.com/containers/podman/pull/18020.  Please reopen if I am mistaken.

**Pull request closed without merging**